rest-java/src/test/java/org/hiero/mirror/restjava/mapper/HookStorageMapperTest.java

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package org.hiero.mirror.restjava.mapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.apache.commons.codec.DecoderException;
+import org.apache.commons.codec.binary.Hex;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class HookStorageMapperTest {
```

Comment:
This needs to become a habit. I'm tired of saying it for every class.

```suggestion
final class HookStorageMapperTest {
```